website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+---
+title: 'React Native 0.71: TypeScript by Default, Flexbox Gap, and more...'
+authors: [lunaleaps, mattcarrollcode, cortinico, kelset]
```

Comment:
historically, we've put the entire release crew as authors of the release blogpost per each release:
- https://reactnative.dev/blog/2022/09/05/version-070
- https://reactnative.dev/blog/2022/06/21/version-069
- https://reactnative.dev/blog/2022/03/30/version-068
- https://reactnative.dev/blog/2022/01/19/version-067

But I understand that a lot of work was done by other folks too. A more realistic authors tag would likely be:
```suggestion
authors: [lunaleaps, mattcarrollcode, cortinico, kelset, dmytrorykun, cipolleschi, titozzz, rickhanlonii]
```

but `rickhanlonii` needs to be added to https://github.com/facebook/react-native-website/blob/main/website/blog/authors.yml

Reply:
Comment from Rick that was on internal:

> I wasn't sure who exactly to put in as the authors, so currently it's [Luna](https://fb.workplace.com/profile.php?id=100029844272183), [Matt](https://fb.workplace.com/profile.php?id=100085011314613), Lorenzo, and [Nicola](https://fb.workplace.com/profile.php?id=100068089718865). In the future I'd like to be able to include more/all authors, or author posts as a team, but with the current design taking up so much vertical space, I think we should limit it to 4 or less people so the content isn't pushed too far down.

Reply:
Added a thanks to the release crew in the Acknowledgements. I'm happy to just be a ghostwriter for this release to cut down on the space the author section takes up.